test_scripts/API/SeparatingChangeAudibleStatus/002_activate_App_audioStreamingState_during_PHONE_CALL_events.lua

```diff
@@ -0,0 +1,322 @@
+        checks = {
+          onHmiStatus = {
+            [1] = { l = "FULL",       a = "AUDIBLE", v = "NOT_STREAMABLE" },
+            [2] = { } -- l = "NONE",  a = "NOT_AUDIBLE", v = "NOT_STREAMABLE"
+          }
+        }
+      },
+      [2] = {
+        action = { event = common.userActions.activateApp, appId = 2 },
+        checks = {
+          onHmiStatus = {
+            [1] = { l = "BACKGROUND", a = "NOT_AUDIBLE",  v = "NOT_STREAMABLE" },
+            [2] = { l = "FULL",       a = "AUDIBLE",      v = "NOT_STREAMABLE" }
+          }
+        }
+      },
+      [3] = {
+        action = { event = common.userActions.phoneCallStart, appId = "none" },
+        checks = {
+          onHmiStatus = {
+            [1] = { }, --l = "BACKGROUND", a = "NOT_AUDIBLE",  v = "NOT_STREAMABLE"
+            [2] = { l = "FULL",       a = "NOT_AUDIBLE",  v = "NOT_STREAMABLE" }
+          }
+        }
+      },
+      [4] = {
+        action = { event = common.userActions.activateApp, appId = 1 },
+        checks = {
+          onHmiStatus = {
+            [1] = { l = "FULL",       a = "NOT_AUDIBLE",  v = "NOT_STREAMABLE" },
+            [2] = { l = "BACKGROUND", a = "NOT_AUDIBLE",  v = "NOT_STREAMABLE" }
+          }
+        }
+      },
+      [5] = {
+        action = { event = common.userActions.phoneCallEnd, appId = "none" },
+        checks = {
+          onHmiStatus = {
+            [1] = { l = "FULL",       a = "AUDIBLE",      v = "NOT_STREAMABLE" },
+            [2] = { } -- l = "BACKGROUND", a = "NOT_AUDIBLE", v = "NOT_STREAMABLE"
+          }
+        }
+      }
+    }
+  }
+}
+
+--[[ Local Functions ]]
+local function doAction(pTC, pStep)
+  pStep.action.event.func(pStep.action.appId)
+  for appId, ohsChecks in ipairs(pStep.checks.onHmiStatus) do
+    common.checkHMIStatus(pTC, pStep.action.event.name, appId, ohsChecks)
+  end
+end
+
+--[[ Scenario ]]
+for n, tc in common.spairs(testCase) do
+  common.Title("TC[" .. string.format("%03d", n) .. "]: "
+    .. "App1[hmiType:" .. tc.apps[1].t .. ", isMedia:" .. tostring(tc.apps[1].m) .. "], "
+    .. "App2[hmiType:" .. tc.apps[2].t .. ", isMedia:" .. tostring(tc.apps[2].m) .. "]")
+  common.Title("Preconditions")
+  common.Step("Clean environment", common.preconditions)
+  common.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+  for appId, params in pairs(tc.apps) do
+    common.Step("Set App " .. appId .. " Config", common.setAppConfig, { appId, params.t, params.m })
+    common.Step("Register App " .. appId, common.registerApp, { appId })
+  end
+
+  common.Title("Test")
+  for i = 1, #tc.steps do
+    common.Step("Action:" .. tc.steps[i].action.event.name .. " app " .. tc.steps[i].action.appId, doAction,
+      { n, tc.steps[i] })
+  end
+
+  common.Title("Postconditions")
+  common.Step("Clean sessions", common.cleanSessions)
+  common.Step("Stop SDL", common.postconditions)
+end
+common.Step("Print failed TCs", common.printFailedTCs)
```

Comment:
Within fix suggested for `common.checkHMIStatus()` script fails in some cases for `Action_Phone_call_end_app_none` step.